test-vectors/did-spec.latest.json

```diff
@@ -2,6 +2,68 @@
   {
     "suite": "did-spec",
     "testResults": [
+      {
+        "ancestors": [
+          "did-spec",
+          "did:key - 2018 Ed25519 cryptosuite - did-key-js - Digital Bazaar",
+          "did-syntax",
+          "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"
+        ],
+        "title": "MUST be a valid URL.",
+        "status": "passed"
+      },
+      {
+        "ancestors": [
+          "did-spec",
+          "did:key - 2018 Ed25519 cryptosuite - did-key-js - Digital Bazaar",
+          "did-syntax",
+          "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"
+        ],
+        "title": "The URI scheme MUST be \"did:\"",
+        "status": "passed"
+      },
+      {
+        "ancestors": [
+          "did-spec",
+          "did:key - 2018 Ed25519 cryptosuite - did-key-js - Digital Bazaar",
+          "did-syntax",
+          "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"
+        ],
+        "title": "The DID method name MUST be an ASCII lowercase string.",
+        "status": "passed"
+      },
+      {
+        "ancestors": [
+          "did-spec",
+          "did:key - 2018 Ed25519 cryptosuite - did-key-js - Digital Bazaar",
+          "did-syntax",
+          "did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH"
+        ],
+        "title": "The DID method name MUST NOT be empty.",
+        "status": "passed"
+      },
+      {
+        "ancestors": [
+          "did-spec",
+          "did:key - 2018 Ed25519 cryptosuite - did-key-js - Digital Bazaar",
+          "5. Core Properties",
```

Comment:
@OR13 is this an issue? I'd like us to group tests by section of the spec, if possible. Make it obvious which tests are testing which part of the spec. Speak clearly to which DID Methods pass/fail which specific statements and make it easy for developers to find these items.

Reply:
I don't think its an issue, but like most software I expect the test suite to change as people try to actually use it.